src/advanced/node-management.md

```diff
@@ -225,33 +218,32 @@ curl http://127.0.0.1:7780/api/system/v1/user_agent
 
 #### Response
 
-Returns string containing information about Exonum, Rust and OS version.
+Returns a string containing information about Exonum, Rust and OS version.
 
-#### Response example
+#### Response Example
 
 ```None
-"exonum 0.6.0/rustc 1.26.0-nightly (2789b067d 2018-03-06)\n\n/Mac OS10.13.3"
+"exonum 0.10.2/rustc 1.32.0 (9fda7c223 2019-01-16)\n\n/Mac OS10.14.3"
 ```
 
-## Private endpoints
+## Private Endpoints
 
-### Add new peer
+### Add New Peer
 
 ```none
 POST {system_base_path}/peers
 ```
 
-Adds new Exonum node to the list of peers for the current node.
-The latter will attempt to connect to the new node asynchronously. If the public
-key of the new node is not in the whitelist, the connection between said nodes
-will not be established.
+Adds new Exonum node to the list of peers for the present node.
+The latter will attempt to [connect](network.md#connect-messages) to the new
+node asynchronously.
 
 #### Parameters
 
-- **address**: PeerAddress
-  IP address of the node which our node have to connect.
-- **public_key**: PublicKey
-  Public key of the other node.
+- **address**: PeerAddress  
+  IP address of the node which the present node should connect to.
+- **public_key**: PublicKey  
+  Public key of the node which the present node should connect to.
 
 #### Example
```

Comment:
Not related to the PR, but still: are we sure this example actually works properly? Backslashes within single quotes do not seem to be processed in any way (at least in bash), thus `--data` parameter is not valid JSON.

Reply:
I have fixed the example